makefile

```diff
@@ -27,40 +27,48 @@ setup-rust:
 aarch64-linux-android: CARGO_ENV_FLAGS += \
 	ISYSROOT=$(NDK_HOME)/toolchains/llvm/prebuilt/linux-x86_64/sysroot \
 	ISYSTEM=$(NDK_HOME)/toolchains/llvm/prebuilt/linux-x86_64/sysroot/usr/include/aarch64-linux-android \
+	AR=llvm-ar \
+	CFLAGS=-Wno-error=unused-but-set-parameter \
```

Comment:
newer NDK uses llvm-ar instead of a platform specific archiver.
Since the use of the archiver is down in C/assembly code this is actually needed for CMake and the previous `CARGO_TARGET_<triple>_AR` was not being passed along to CMake